tox.ini

```diff
@@ -17,7 +17,7 @@ deps =
     pytest-flakefinder
 install_command = 
     py{27,35,36,37,38,39}: pip install -f https://download.pytorch.org/whl/torch_stable.html {opts} {packages}
-    pylaunch: pip install -f https://download.pytorch.org/whl/torch_stable.html docker chardet iso8601 jupyter-repo2docker {opts} {packages}
+    pylaunch: pip install -f https://download.pytorch.org/whl/torch_stable.html defusedxml mistune testpath jupyterlab_pygments pandocfilters nbconvert nbformat docker chardet iso8601 jupyter-repo2docker {opts} {packages}
```

Comment:
This sucks. They're needed for tests to pass. nbconvert requires all of these to run.

Reply:
That can't be right, they must all be dependencies of `nbconvert` no?  It may be that {opts} is adding a `--no-deps` flag?  I think you can just add `.[launch]` to the last line in requirements_dev.txt and we'll have everything we need...